### DISCUSSION TITLE
Configure image formats > Scale / Crop > v1.4 vs. 2.4

### DISCUSSION THREAD

**Discussion by normenrollf** [sulu/sulu#6543] [Q&A] [answered]
In version 1.4, it was possible to crop and scale images individually. Do any of you know why this function was eliminated and/or whether and how the commands still work in V2.4? 

https://docs.sulu.io/en/1.4/book/image-formats.html?highlight=image%20format

**Comment by ajdinmore** [comment]
Don't know if there's a way to automatically set crops, but they can be made on a per-image basis in the admin on 2.4 which usually makes more sense as the content editor then has control over it:
https://sulu.rocks/admin/#/media/en/26/details

![image](https://user-images.githubusercontent.com/5090062/164024060-21162589-346f-4b33-8695-96b8125299d0.png)

**Reply by normenrollf** [reply]
Hi Andrew, 

thank you for your link and advice. I did not know this function until now. Are the default settings to be found in the file image-formats.xml or is there another file for this?

My original question was aimed at a different problem. We need an aspect ratio of 4:3.7 because we want to mask an image and the mask is the customer logo. We can upload the images in this aspect ratio ... but more contemporary would be a preset in the image-formats.xml file that would automate this. Then the crop function you mentioned with the stored aspect ratio would be the icing on the cake.

**Reply by ajdinmore** [reply]
`image-formats.xml` is where you would define the dimensions of the crop you need, yes, though the schema is different with 2.4.  Details on using the defined formats are at the end of the docs page.

https://docs.sulu.io/en/2.4/book/image-formats.html

**Comment by niklasnatter** [comment, marked as answer]
Hey, if you define your image format in the configuration of your project, Sulu will automatically generate an image with the configured dimension on the first request to the format: https://docs.sulu.io/en/2.4/book/image-formats.html

If the ratio of the dimensions dont match the uploaded image, Sulu will use a crop of the center of the image by default. The content manager can influence this crop in the administration interface by defining it explicitly or setting a focus point.
